index.html

```diff
@@ -34,12 +34,236 @@ <h3>I am a software developer</h3>
+                                    <ul class="tags">
+                                        <li class="tags1"><h6>html</h6></li>                                            
+                                        <li class="tags2"><h6>bootstrap</h6></li>
+                                        <li class="tags3"><h6>Ruby</h6></li>
+                                    </ul>
+                                </section>
+                            </section>
+                        </section>
+                        <!-- <section class="mask-sub"></section> -->
+                    </section>
+                    <section class="action">
+                        <section class="but">
+                            <p class="see1">See Project</p>
+                        </section>
+                    </section>
+                </section>
+            </section>
+            <section class="about">
+                <p class="primary">About Me</p>
+                <p class="description">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                <section class="img-about"><img src="images/illustration-about-me-mobile.png" alt=""></section>
+                <section class="button">
+                    <p class="get">Get My Resume</p>
+                </section>
+                <section class="divider">
+                    <div class="divi"></div>
+                </section>
+                <section class="skill">
+                    <ul >
+                        <li class="li-items">
+                            <div class="li-main">
+                                <div class="tri"></div>
+                                <p>languages</p>
+                            </div>
+                            <div>
+                                <ul class="lan">
+                                    <div class="mjava"><li class="java">Javascript</li></div>
+                                    <div class="mruby"><li class="ruby">Ruby</li></div>
+                                    <div class="mhtml"><li class="html">html</li></div>
+                                    <div class="mcss"><li class="css">css</li></div>
+                                </ul>
+                            </div>
+                        </li>
+                        <li class="li-items1">
+                            <div class="li-main">
+                                <div class="squ"></div>
+                                <p>Frameworks</p>
+                            </div>
+                            <div>
+                                <ul class="lan">
+                                    <div class="mjava"><li class="java">bootstrap</li></div>
```

Comment:
- [ ] **[OPTIONAL]** Please 🙏🏽 be careful with the orthography for a better user interface.  The correct one is "Bootstrap".